src/main/java/no/nav/folketrygdloven/beregningsgrunnlag/ytelse/svp/FinnGrenseverdi.java

```diff
@@ -60,15 +68,37 @@ public Evaluation evaluate(BeregningsgrunnlagPeriode grunnlag) {
 				resultater.put("inntektgraderingsprosent", grunnlag.getInntektsgraderingFraBruttoBeregningsgrunnlag());
 			}
 		}
-		resultater.put("grenseverdi", grenseverdi);
-		grunnlag.setGrenseverdi(grenseverdi);
-		SingleEvaluation resultat = ja();
-		resultat.setEvaluationProperties(resultater);
-		return resultat;
+		return grenseverdi;
+	}
 
+	private static BigDecimal utførMidlertidigInaktivGradering(BeregningsgrunnlagPeriode grunnlag, Map<String, Object> resultater, BigDecimal grenseverdi) {
+		//hvis §8-47a, skaler med fast faktor
+		var erInaktivTypeA = MidlertidigInaktivType.A.equals(grunnlag.getBeregningsgrunnlag().getMidlertidigInaktivType());
+		if (erInaktivTypeA) {
+			BigDecimal reduksjonsfaktor = grunnlag.getBeregningsgrunnlag().getMidlertidigInaktivTypeAReduksjonsfaktor();
+			grenseverdi = grenseverdi.multiply(reduksjonsfaktor);
+			resultater.put("grad847a", reduksjonsfaktor);
+		}
+		return grenseverdi;
+	}
+
+	private static BigDecimal utførYtelsesspesifikkGradering(BeregningsgrunnlagPeriode grunnlag, BigDecimal grenseverdi) {
+		if (grunnlag.getBeregningsgrunnlag().getYtelsesSpesifiktGrunnlag() != null && grunnlag.getBeregningsgrunnlag().getYtelsesSpesifiktGrunnlag() instanceof PleiepengerSyktBarnGrunnlag psbGrunnlag) {
+			var tilsynsgraderingsprosent = psbGrunnlag.getTilsynsgraderingsprosent().compress((b1, b2) -> b1.compareTo(b2) == 0, StandardCombinators::leftOnly);
+			var intersection = tilsynsgraderingsprosent.intersection(new LocalDateInterval(grunnlag.getPeriodeFom(), grunnlag.getPeriodeTom()));
+			if (intersection.toSegments().size() > 1) {
+				throw new IllegalStateException("Fant flere tilsynsgrader for samme periode " + grunnlag.getPeriodeFom());
+			}
+			if (!intersection.isEmpty()) {
+				var tilsynsgradering = intersection.toSegments().first().getValue();
+				grenseverdi = grenseverdi.multiply(tilsynsgradering.divide(BigDecimal.valueOf(100), 10, RoundingMode.HALF_UP));
```

Comment:
Mulig jeg misforstod, men ender vi opp med å gradere mot tilsyn to ganger her? Det er vel også inkludert i utbetalingsgradene fra uttak som brukes over?

Reply:
Eg trudde vi blei enige om å sende inn 1-aktivitetsgrad som utbetalingsgrad.